khal/settings/utils.py

```diff
@@ -57,6 +57,21 @@ def is_timedelta(string):
     except ValueError:
         raise VdtValueError("Invalid timedelta: {}".format(string))
 
+def is_theme(option):
+    """checks if *theme* is a valid value
+
+    :param option: the option the user set in the config file
+    :type option: str
+    :returns: user
+    :rtype: str
+    """
```

Comment:
Use type hints instead:

```suggestion
def is_theme(option: str) -> str:
    """checks if *theme* is a valid value

    :param option: the option the user set in the config file
    :returns: user
    """
```